app/templates/view_users.html

```diff
@@ -0,0 +1,163 @@
+{% import "toolkit/summary-table.html" as summary %}
+{% from 'macros/breadcrumb.html' import breadcrumb as breadcrumb %}
+{% from 'macros/page_heading.html' import page_heading %}
+
+{% extends "_base_page.html" %}
+
+{% block page_title %}
+    Users – Digital Marketplace admin
+{% endblock %}
+
+{% block content %}
+{%
+    with items = [
+        {
+            "link": url_for('.index'),
+            "label": "Admin home"
+        },
+    ]
+%}
+{% include "toolkit/breadcrumb.html" %}
+{% endwith %}
+
+<div class="page-container">
+
+
+    {% with messages = get_flashed_messages(with_categories=true) %}
+    {% if messages %}
+        {% for category, message in messages %}
+            {% if message == 'no_users' %}
+                {% set displayed_message = "Sorry, we couldn't find an account with that email address" %}
+            {% endif %}
+            {%
+            with
+                message = displayed_message,
+                type = "destructive" if category == 'error' else "success"
+            %}
+            {% include "toolkit/notification-banner.html" %}
+            {% endwith %}
+        {% endfor %}
+    {% endif %}
+    {% endwith %}
+
+    {% with heading = email_address if email_address %}
+    {% include "toolkit/page-heading.html" %}
+    {% endwith %}
+
+    {% call(item) summary.list_table(
+            users,
+            caption="Users",
+            empty_message="No users to show",
+            field_headings=[
+                'Name',
+                'Role',
+                'Supplier',
+                'Last login',
+                'Last password change',
+                'Locked',
+                summary.hidden_field_heading("Change status")
+            ],
+            field_headings_visible=True)
+        %}
+        {% call summary.row() %}
+
+        {{ summary.field_name(item.name) }}
+
+        {{ summary.field_name(item.role) }}
```

Comment:
In terms of the wide `Role` column, I think it's because it's getting `class=summary-item-field-first` (as does `name`) - but it shouldn't have the `first` on the end.  I have no idea how you'd change this though.